src/markdown.cpp

```diff
@@ -59,7 +59,9 @@
 #ifdef ENABLE_TRACING
 #define IOSTREAM      stdout
 #define DATA_BUFSIZE  20
-#if defined(_WIN32) && !defined(CYGWIN)
+#ifdef __MINGW32__
+#define PRETTY_FUNC __PRETTY_FUNCTION__
+#elif defined(_WIN32) && !defined(CYGWIN)
```

Comment:
Why not use:
```
#if defined(_WIN32) && !defined(CYGWIN) && !defined (__MINGW32__)
```
instead of the original:
```
#if defined(_WIN32) && !defined(CYGWIN)
```

Reply:
Hi, no particular reason in fact you're right this version is more compact, changed  in new commit

Reply:
Looks like when correcting the `#if` construct by accident you pasted `#ifdef` instead `#if`, now the fix reads:
```
#ifdef defined(_WIN32) && !defined(CYGWIN) && !defined(__MINGW32__)
```
instead of the intended:
```
#if defined(_WIN32) && !defined(CYGWIN) && !defined(__MINGW32__)
```